[ONNX] Remove f arg from export_to_pretty_string (#69045)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #69550
* #69549
* #69548
* #69547
* __->__ #69546
* #69544
* #68491
* #68490

The arg is not used and was previously deprecated.

Also remove torch.onnx._export_to_pretty_string. It's redundant with the
public version.

Differential Revision: [D32994270](https://our.internmc.facebook.com/intern/diff/D32994270)

BowenBao committed:
[ONNX] Remove f arg from export_to_pretty_string (#69045)

The arg is not used and was previously deprecated.

Also remove torch.onnx._export_to_pretty_string. It's redundant with the
public version.

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32994270).

BowenBao committed:
Update on "[ONNX] Remove f arg from export_to_pretty_string (#69045)"


The arg is not used and was previously deprecated.

Also remove torch.onnx._export_to_pretty_string. It's redundant with the
public version.

Differential Revision: [D32994270](https://our.internmc.facebook.com/intern/diff/D32994270)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32994270).

BowenBao committed:
Update on "[ONNX] Remove f arg from export_to_pretty_string (#69045)"


The arg is not used and was previously deprecated.

Also remove torch.onnx._export_to_pretty_string. It's redundant with the
public version.

Differential Revision: [D32994270](https://our.internmc.facebook.com/intern/diff/D32994270)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32994270).
